### ISSUE TITLE
Training on 2014 + 2017 COCO doesn't make sense

### ISSUE THREAD

**Issue by HamsterHuey** [vietnguyen91/Yolo-v2-pytorch#6]
Nice work with the repo. However, I noticed that you seemed to have combined the 2014 and 2017 data to train. However, if you read the COCO website carefully, you will find that the 2014 and 2017 data are identical (exactly the same number of total images). It's just that in 2017, they partitioned most of the images to be in train and only 5000 to be in the validation set.

If you train on 2014 train+val and 2017 train, you have essentially trained on the test data, so all metrics at that point are meaningless.

This is exactly the reason why your "test" set loss plots for 2014+2017 data show the test loss to continue to go down with epochs without ever entering a regime of being overtrained to your test dataset.

**Comment by vietnguyen91** [comment]
You are right. At the time I coded this project, I didnt know it. I forgot to put it out. Thanks for reminding
